fix(windows): make local account passwords never expire

lvillani opened the pull request:
# Description

Set unlimited password age for local accounts on Windows.

Encountered this issue on a couple of long-lived Windows 10 22H2 VMs and tested the fix there. The fix should apply also to recent Windows 11 builds but haven't had a chance to test there yet.

Thanks to user @bogiord for providing the fix. I'm just packaging it up in a PR 😃 

- Fixes #1622 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have tested my code in common scenarios and confirmed there are no regressions
- [x] I have added comments to my code, particularly in hard-to-understand sections

philclifford reviewed (changes requested):
Needs a little conflict resolution.

You might like to [add a trailer](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) to your commit message to reflect your expressed thanks to Bogdan.

lvillani commented:
Rebased, solved conflicts and added a trailer to the commit message. I also had the chance to test these changes on Windows 11 25H2 (I have updated the description to match).
